go/libkb/secret_store.go

```diff
@@ -115,7 +118,7 @@ func GetConfiguredAccounts(c SecretStoreContext, s SecretStoreAll) ([]keybase1.C
 	for _, username := range storedSecretUsernames {
 		nu := NewNormalizedUsername(username)
 		account, ok := accounts[nu]
-		if ok {
+		if ok && !nu.IsNil() {
```

Comment:
Let's get rid of this check too. I don't see how we could have an empty username here.